### DISCUSSION TITLE
How to style the Popover element of the DatePickerInput component? (v. 7.1.5)

### DISCUSSION THREAD

**Discussion by dmitry-ff** [mantinedev/mantine#5112] [Q&A]
How to style the Popover element of the DatePickerInput component? (v. 7.1.5)

**Comment by rtivital** [comment]
Use `popoverProps={{ classNames: yourClassNames }}`, you can learn more about styling on the Styles API page.

**Reply by dmitry-ff** [reply]
Thanks for the explanation. Is it possible to somehow change the way the datePicker value is set (for example, when selecting a date and clicking the "apply" button)

**Reply by rtivital** [reply]
You can build any custom inputs using Popover and DatePicker components as a base.
